hack/lib/golang.sh

```diff
@@ -496,7 +496,8 @@ kube::golang::outfile_for_binary() {
 # Returns 0 if the binary can be built with coverage, 1 otherwise.
 # NB: this ignores whether coverage is globally enabled or not.
 kube::golang::is_instrumented_package() {
-  return "$(kube::util::array_contains "$1" "${KUBE_COVERAGE_INSTRUMENTED_PACKAGES[@]}")"
+  kube::util::array_contains "$1" "${KUBE_COVERAGE_INSTRUMENTED_PACKAGES[@]}"
+  return $?
```

Comment:
FYI @Katharine I'm having one of those "how did this ever work at all?" moments...

We used to return the output of `array_contains`, not its exit code.  But `array_contains` never outputs anything, it just returns.

Reply:
This worked because in the absence of any other argument `return` returns the exit code of the last command. Since `$(kube::util::array_contains)` evaluated to an empty string and was unquoted, `return` effectively had no arguments. The subshell correctly propagates the exit code from its last command as its own exit code, and so the effect was equivalent to calling `$(kube::util::array_contains)`, followed by `return`.